doc/api/tls.md

```diff
@@ -2,90 +2,138 @@
 
     Stability: 2 - Stable
 
-Use `require('tls')` to access this module.
+The `tls` module provides an implementation of the Transport Layer Security
+(TLS) and Secure Socket Layer (SSL) protocols that is built on top of OpenSSL.
```

Comment:
I think we do not support either of `SSL2` or `SSL3` by default anymore. Is it worth mentioning here?